test/run_tests.sh

```diff
@@ -183,7 +193,10 @@ function run_mp_op_tests {
 }
 
 function run_tests {
-  run_op_tests
+  if ["$PYTORCH_XLA_TESTS_ONLY" = false] 
+    run_op_tests_py
```

Comment:
NB: Can we say `pt` and `ptxla` instead of `py` and `pyxla` for these function names?

Reply:
sg, will replace after this trigger result

Reply:
Thanks for the comments, Steven, updated in commit [e3d48d4](https://github.com/pytorch/xla/pull/4444/commits/e3d48d422c6eb65b941880073c3cf92e0292ed8f)